extensions/amp-lightbox/0.1/amp-lightbox.css

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+amp-lightbox {
+    position: fixed !important;
+    z-index: 1000 !important;
+    top: 0 !important;
+    left: 0 !important;
+    bottom: 0 !important;
+    right: 0 !important;
```

Comment:
add `display:none` (without !important)

Reasons is, the old code used to only apply this style first time lightbox-opened, now this gets applied as soon as page loads. Technically, `<amp-lightbox>` required to have `layout=nodisplay` to be valid AMP BUT we should not break the world for non-valid AMP either (this will effectively make the whole page unclickable if they forget to add `layout=nodisplay`)